app/src/main/res/xml/logins_preferences.xml

```diff
@@ -2,7 +2,7 @@
 <!-- This Source Code Form is subject to the terms of the Mozilla Public
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
-<PreferenceScreen xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.preference.PreferenceScreen xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Curious why we add androidx.preference?

Reply:
🤔 Unsure. I probably used something from androidx and forgot to change it back 🤷 
I'll change it back.